Automattic-Tracks-iOS/Event Logging/EventLoggingDataSource.swift

```diff
@@ -9,11 +9,22 @@ public protocol EventLoggingDataSource {
 
     /// The path to the log file for the most recent session.
     var previousSessionLogPath: URL? { get }
+
+    /// The path to the log file for the current session.
+    var currentSessionLogPath: URL? { get }
```

Comment:
Yeah, this is a really subtle and mildly annoying thing. This depends on how the hosting app implements logging.

The WordPress app implements logging by day, so each day has its own log file. What would IMHO make a little more sense is a per-session model – so every session has its own log file (this would make it far easier to separate data based on when a crash happened).

In a per-session model, you'd send the current session's log file for developer-requested events (ie – please send this exception to Sentry – you want to know what happened during the session) and the previous session's log file for crash events (because now the current session log file is empty – you're in a new one after first launch from a crash, so you want the previous one).

We could change the `previousSessionLogPath` name to `logFilePathForMostRecentlyCrashedSession` or something similar? WDYT? I went back and forth on this a whole bunch.

Reply:
Thank you for the explanation. Maybe we add some comments explaining this, but it doesn't look like we can improve this easily, so for now I am OK with leaving it as is. We can re-visit it later.